src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Formatting/FormattingPassBase.cs

```diff
@@ -621,6 +638,18 @@ bool IsImplicitOrExplicitExpression()
                 //
                 return owner.AncestorsAndSelf().Any(n => n is CSharpImplicitExpressionSyntax || n is CSharpExplicitExpressionSyntax);
             }
+
+            bool IsInSectionDirective()
+            {
+                var directive = owner.FirstAncestorOrSelf<RazorDirectiveSyntax>();
+                if (directive != null &&
+                    directive.DirectiveDescriptor.Directive == SectionDirective.Directive.Directive)
+                {
+                    return true;
+                }
+
+                return false;
```

Comment:
Nit;

```suggestion
                return directive != null &&
                    directive.DirectiveDescriptor.Directive == SectionDirective.Directive.Directive;
```

granted this is more so a personal preference and I can see how the current presentation could be clearer, so I'm 💯  okay to leave as is.

Reply:
This is a trade off so we can have a better debugging experience. Saves us from hovering over the values to determine whether this returns true or false.

Reply:
I say this but I just looked at all of the other methods above this and they all don't follow this 🤦‍♂️.